misk/src/main/kotlin/misk/client/HttpClientConfig.kt

```diff
@@ -0,0 +1,61 @@
+package misk.client
+
+import misk.config.Config
+import misk.security.ssl.KeystoreConfig
+import misk.security.ssl.SslContextFactory
+import okhttp3.OkHttpClient
+import java.net.Proxy
+import java.time.Duration
+import java.util.concurrent.TimeUnit
+import javax.net.ssl.X509TrustManager
+
+data class HttpClientConfig(
+        private val defaultConnectTimeout: Duration? = null,
```

Comment:
it might be nice to document what does `null` mean for these params. No timeout?